samples/cellular/modem_shell/prj.conf

```diff
@@ -129,6 +129,7 @@ CONFIG_LTE_LC_TAU_PRE_WARNING_MODULE=y
 CONFIG_DATE_TIME=y
 
 # nRF Cloud A-GNSS, disable this in case you want to enable SUPL below
+CONFIG_NRF_CLOUD=y
```

Comment:
I'll mark a comment here as MOSH CI seems to fail. Please have a look @moksanen.